README.md

```diff
@@ -72,6 +72,11 @@ add_filter(
 Changelog
 ---------
 
+### 0.5.2
+
+* **POSSIBLE BREAKING CHANGE** Refactors `SP_Heartbeat` to store the last time the beat was checked as well as the last time the beat was verified. `SP_Heartbeat::get_last_beat()` previously returned an int, but now returns an array.
```

Comment:
I'm assuming the possible work around here would be to manually delete this option value?

Reply:
I'm not following -- do you mean what a plugin/theme currently using `SP_Heartbeat::get_last_beat()` should do to work around the change?

Reply:
correct!